### PR TITLE
create task_9

### PR DESCRIPTION
**Pull request by TanyaLevchenya**

### CODE REVIEW

File: TrainingsSite/style.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+           color: #8abbe6;
+           height: 40px;
+           line-height: 40px;
+           width: 21%;
+           float: left;
+           padding-left: 20px;
+           margin: 5px;
+           min-width: 255px;
+}
+div > div:hover{ box-shadow: -1px 6px 16px 1px rgba(5, 5, 5, 0.925);
```
Comment (Killerkip10) [inline TrainingsSite/style.css]: Oh no)))
`.class > div:hover`

File: TrainingsSite/style.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+}
+h2{ text-transform: uppercase; 
+    text-align: center;
+    color: #252525;
+}
+.menu{ padding-bottom: 25px;
+}
+.clear{ clear: both;
+}
+div > div{ background-color: white;
```
Comment (Killerkip10) [inline TrainingsSite/style.css]: This selector is too common.

File: TrainingsSite/style.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+       font-family: Roboto, serif;
+}
+header{ background-color: black; 
+        color: white; 
+        height: 35px; 
+        line-height: 20px;
+        font-size: 20px;
+        padding-left: 20px;
+}
+span{ padding-left: 35px;
```
Comment (Killerkip10) [inline TrainingsSite/style.css]: All span in your page should have padding-left ?)

File: TrainingsSite/style.css [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+}
+a{ text-decoration: none;
+   font-size: 15px;
+   color: white;
+   padding-left: 10px;
```
Comment (Killerkip10) [inline TrainingsSite/style.css]: Okey, text-decoration, font-size, color you can set here, but padding-left. Are sure that all links on your page should have padding-left?) I think it is the single case

File: TrainingsSite/style.css [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+         display: flex;
+         align-items: center;
+         background-repeat: no-repeat;
+         background-position: center center;
+         background-attachment: fixed;
+         background-size: cover;
+}
+h3{ color: white;
+    font-size: 55px;
+    padding-left: 150px;
```
Comment (Killerkip10) [inline TrainingsSite/style.css:34]: Also

File: TrainingsSite/style.css [hunk 6]
```diff
@@ -0,0 +1,141 @@
+body{ margin: 0;
```
Comment (Killerkip10) [inline TrainingsSite/style.css:1]: Please align all  your css code

File: TrainingsSite/style.css [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+    color: #4c96d8; 
+    cursor: pointer;
+}     
+.left_cell div:first-child:hover{ 
+    color: #151d3d;
+}
+.left_cell div:last-child{
+    font-size: 15px;
+}
+.middle_cell div:nth-child(-n+3){
```
Comment (Killerkip10) [inline TrainingsSite/style.css]: :nth-child(-n+3) is redundant))

File: TrainingsSite/index.html [hunk 8]
```diff
@@ -5,143 +5,95 @@
...
+        <td class="middle_cell">
+            <div><span>755 BYN</span></div>
+            <div>128 уч.ч (три месяца)</div>
+            <div><span>Осталось</span>10 из 20 мест</div>
         </td>
-        <td style="width: 180px; padding-right: 15px;">
-            <div class="button">Записаться</div>
-            <div class="key">Программа курса</div>
+
+        <td class="button">
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: It is not a button)
button inside

File: TrainingsSite/style.css [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+         background-repeat: no-repeat;
+         background-position: center center;
+         background-attachment: fixed;
+         background-size: cover;
+}
+h3{ color: white;
+    font-size: 55px;
+    padding-left: 150px;
+}
+@media (max-width: 800px){
```
Comment (Killerkip10) [inline TrainingsSite/style.css]: Please move all @media down

File: TrainingsSite/style.css [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+    font-size: 15px;
+}
+.middle_cell div:nth-child(-n+3){
+    padding: 5px;
+}
+.button{
+    width: 190px;
+    padding-right: 15px;
+}
+.button div:first-child{
```
Comment (Killerkip10) [inline TrainingsSite/style.css]: .button div:first-child and .button div:last-child have the same code
Please create class .button with common css prop.

File: TrainingsSite/index.html [hunk 11]
```diff
@@ -5,143 +5,95 @@
...
+            <div>01.09.2020-30.12.2020</div>
+            <div>Занятия вторник и пятница с 18.30 до 21.20</div>
         </td>
-        <td>
-            <div style="padding: 4px;"><strong>859 BYN</strong></div>
-            <div style="font-size: 15px; padding: 4px;">120 уч.ч (три месяца)</div>
-            <div style="font-size: 12px; padding: 4px;"><span style="font-weight: bold;">Осталось</span> 8 из 12 мест</div>
+
+        <td class="middle_cell">
+            <div><span>859 BYN</span></div>
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: I think this `span` is redundant. Also fix below

**Review by Killerkip10 — COMMENTED** [review]
## Homework 9

### Total 100 points

File: TrainingsSite/style.css [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+    h3{
+       padding-left: 0; 
+       text-align: center;}
+}
+@media (max-width: 320px){
+    section{ 
+      justify-content: flex-start;
+      display: flex;
+      justify-content: center;
+      background-image: url(./pexels-photo-583846.jpg);}
```
Comment (Killerkip10) [inline TrainingsSite/style.css]: Picture is too blurry

File: TrainingsSite/style.css (hunk 2 above)
Comment (TanyaLevchenya) [reply TrainingsSite/style.css]: corrected

File: TrainingsSite/style.css (hunk 7 above)
Comment (TanyaLevchenya) [reply TrainingsSite/style.css]: ok

File: TrainingsSite/index.html (hunk 8 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: ok

File: TrainingsSite/style.css (hunk 9 above)
Comment (TanyaLevchenya) [reply TrainingsSite/style.css]: correct

File: TrainingsSite/style.css (hunk 10 above)
Comment (TanyaLevchenya) [reply TrainingsSite/style.css]: correct

File: TrainingsSite/style.css (hunk 1 above)
Comment (TanyaLevchenya) [reply TrainingsSite/style.css]: ok

File: TrainingsSite/style.css (hunk 12 above)
Comment (TanyaLevchenya) [reply TrainingsSite/style.css]: correct

File: TrainingsSite/style.css (hunk 3 above)
Comment (TanyaLevchenya) [reply TrainingsSite/style.css]: да

File: TrainingsSite/style.css (hunk 4 above)
Comment (TanyaLevchenya) [reply TrainingsSite/style.css]: да

File: TrainingsSite/style.css (hunk 5 above)
Comment (TanyaLevchenya) [reply TrainingsSite/style.css:34]: да

File: TrainingsSite/index.html (hunk 11 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: да

**Review by Killerkip10 — COMMENTED** [review]
## Task 9
### Total 100 points

**Pull request merged**